packages/@sanity/form-builder/src/inputs/PortableText/text/AnnotationToolbarPopover.tsx

```diff
@@ -59,7 +59,7 @@ export function AnnotationToolbarPopover(props: AnnotationToolbarPopoverProps) {
         return cursorRect
       },
     }
-  }, [cursorRect])
+  }, [cursorRect]) as HTMLElement
```

Comment:
Potential cause of error if downstream code paths expect this to have the full HTMLElement API. It's also inaccurate since it may return `null`.

Could the return type of the memoized function instead be set to `null | {getboundignClientRect: HTMLElement['getBoundingClientRect']}`?